app/services/events/record.rb

```diff
@@ -170,35 +189,25 @@ def self.teacher_trs_attributes_updated!(author:, teacher:, modifications:, happ
       new(event_type:, author:, modifications:, teacher:, heading:, happened_at:).record_event!
     end
 
-    # Admin events
+    # Induction Extension Events
 
-    def self.record_induction_period_updated!(author:, modifications:, induction_period:, teacher:, appropriate_body:, happened_at: Time.zone.now)
-      event_type = :induction_period_updated
-
-      heading = 'Induction period updated by admin'
+    def self.record_appropriate_body_adds_induction_extension_event(author:, appropriate_body:, teacher:, induction_extension:, modifications:, happened_at: Time.zone.now)
+      event_type = :appropriate_body_adds_induction_extension
+      teacher_name = Teachers::Name.new(teacher).full_name
+      heading = "#{teacher_name}'s induction extended by #{induction_extension.number_of_terms} terms"
 
-      new(event_type:, modifications:, author:, appropriate_body:, induction_period:, teacher:, heading:, happened_at:).record_event!
+      new(event_type:, author:, appropriate_body:, teacher:, induction_extension:, modifications:, heading:, happened_at:).record_event!
     end
 
-    def self.record_admin_passes_teacher_event(author:, appropriate_body:, induction_period:, teacher:)
-      fail(NoInductionPeriod) unless induction_period
-
-      event_type = :teacher_passes_induction
-      heading = "#{Teachers::Name.new(teacher).full_name} passed induction (admin)"
-      happened_at = induction_period.finished_on
+    def self.record_appropriate_body_updates_induction_extension_event(author:, appropriate_body:, teacher:, induction_extension:, modifications:, happened_at: Time.zone.now)
+      event_type = :appropriate_body_updates_induction_extension
+      teacher_name = Teachers::Name.new(teacher).full_name
+      heading = "#{teacher_name}'s induction extended by #{induction_extension.number_of_terms} terms"
 
-      new(event_type:, author:, appropriate_body:, teacher:, induction_period:, heading:, happened_at:).record_event!
+      new(event_type:, author:, appropriate_body:, teacher:, induction_extension:, modifications:, heading:, happened_at:).record_event!
     end
 
-    def self.record_admin_fails_teacher_event(author:, appropriate_body:, induction_period:, teacher:)
-      fail(NoInductionPeriod) unless induction_period
-
-      event_type = :teacher_fails_induction
-      heading = "#{Teachers::Name.new(teacher).full_name} failed induction (admin)"
-      happened_at = induction_period.finished_on
-
-      new(event_type:, author:, appropriate_body:, teacher:, induction_period:, heading:, happened_at:).record_event!
-    end
+    # Admin Events
 
     def self.record_teacher_induction_status_reset_event!(author:, appropriate_body:, teacher:)
```

Comment:
Is this event one for the `Teacher Status Events` section above, or is it different, or deserve its own section?